samples/subsys/usb/mass/boards/stm32l562e_dk_ns.overlay

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2022 O.S.Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		zephyr,code-partition = &slot1_partition;
+	};
+};
+
+&flash0 {
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x00000000 0x00019000>;
+			read-only;
+		};
+		/* Secure image primary slot */
+		slot0_partition: partition@19000 {
+			label = "image-0";
+			reg = <0x00019000 0x0003C000>;
+		};
+		/* Non-secure image primary slot */
+		slot1_partition: partition@55000 {
+			label = "image-1";
+			reg = <0x00055000 0x0002A000>;
+		};
```

Comment:
Can this be moved to board description ?

Reply:
Done!